script.js

```diff
@@ -1,4 +1,32 @@
+var winCount = 0;
+var lostCount = 0;
+var drawCount = 0;
+var count = 0;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  ++count;
+  if ((input == 'scissors') || (input == 'stone') || (input == 'paper')) {
+    var myOutputValue = getResult(input);
+    return myOutputValue;
+  };
+  return "The input options are scissors, stone or paper. Kindly select from these";
 };
+
+var getResult = function (input) {
+  console.log('you have input ' + input);
+  var handArray = ['stone', 'paper', 'scissors'];
+  var randomInteger = Math.floor(Math.random() * 3);
+  var chosenHand = handArray[randomInteger];
+  console.log('system has chosen ' + chosenHand);
+  if (input == chosenHand) {
+    ++drawCount;
+    return ('System also chose ' + chosenHand + '. Its a Draw. No. of draws = ' + drawCount + '/' + count);
+
+  };
+  if (((input == 'scissors') && (chosenHand == 'paper')) ||
```

Comment:
Very nice that you've combined the conditions for win, lose and draw respectively. Especially that the lose condition is the default case if there's no win or draw.